src/react/databrowser/buckets/details/Overview.tsx

```diff
@@ -97,14 +96,43 @@ function Overview({ bucket, ingestionStates }: Props) {
   const workflowsQuery = useWorkflows();
   const features = useSelector((state: AppState) => state.auth.config.features);
   const { account } = useCurrentAccount();
+  const [isErrorModalOpen, setIsErrorModalOpen] = useState(false);
+  const [attachedWorkflowsCount, setAttachedWorkflowsCount] = useState(0);
```

Comment:
Not sure we need a state for this count. I think we could get the number the workflow directly given the bucket name.